src/inputs/Table/components/TableToolbar/style.js

```diff
@@ -8,12 +8,20 @@ export const TABLE_TOOLBAR_HEIGHT = '32px';
 export const useStyles = makeStyles((theme) => ({
   tableToolbar: {
     border: 'var(--ag-borders) var(--ag-border-color)',
-    borderBottom: 'none',
-    padding: theme.spacing(1, 2),
+    padding: theme.spacing(0.2, 0),
     width: '100%',
     height: TABLE_TOOLBAR_HEIGHT,
     color: theme.palette.primary.main,
     backgroundColor: 'var(--ag-odd-row-background-color)',
-    fontSize: '16px',
+    '& .MuiButtonBase-root': {
+      marginLeft: theme.spacing(1),
+    },
+  },
+  TableLoading: {
```

Comment:
Can you rename this key to use a lowercase "t"?

Reply:
DONE